<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSegmentPreview</class>
 <widget class="QWidget" name="DialogSegmentPreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DialogSegmentPreview</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>401</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGraphicsView" name="regImageColorretView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>401</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QSlider" name="expectedPatchesNumSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>预期分割数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PreviewSegmentImageBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>预览</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="expectedPatchesNumspinBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="OKSegmentPreviewImageBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
